Tools/TestWebKitAPI/WebTransportServer.mm

```diff
@@ -40,42 +40,56 @@
 
     Function<Task(Connection)> connectionHandler;
     RetainPtr<nw_listener_t> listener;
-    Vector<RetainPtr<nw_connection_group_t>> connectionGroups;
```

Comment:
I really don't think moving away from nw_connection_group_t is a good idea.  Can we not just make nw_connection_group_t work in Network.framework instead?